test/api/python/finite_field.py

```diff
@@ -0,0 +1,54 @@
+###############################################################################
+# Top contributors (to current version):
+#   Andrew Reynolds, Alex Ozdemir
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+#
+# A simple start-up/tear-down test for cvc5.
+#
+# This simple test just makes sure that the public interface is
+# minimally functional.  It is useful as a template to use for other
+# system tests.
+##
+
+import cvc5
```

Comment:
This test is broken without configuring with `--cocoa`. Should I guard it? How can I even do that in Python? I guess I'd need to guard the test in CMake?

Reply:
You were able to fix this in the CMake, right?

Reply:
Yep!